sporkfish/lichess_bot/lichess_bot_berserk.py

```diff
@@ -144,13 +145,39 @@ def set_pos_and_play_move(
                 num_moves = len(state["moves"].split())
                 prev_moves = state["moves"]
                 set_pos_and_play_move(num_moves, color, prev_moves, game_id, state)
+            elif state["type"] == "opponentGone" and state["gone"]:
+                time.sleep(8)
```

Comment:
I have a sneaky suspicion that this doesn't work for all cases. Have you tested? IIRC taking a win when opponent gone in rapid is at least 30 seconds.

Btw I'd suggest (once you confirm that 8 is the correct number) to add this as a class level variable with a suitable name (`_claim_victory_opponent_gone_after_seconds` or if you think of better, plus maybe a comment). Other people will have no idea why there's a magic number 8 here, or why we are waiting at all to claim a victory.

Reply:
Wait, I just read the API. https://lichess-org.github.io/berserk/api.html#berserk.clients.Board This is for external (physical electronic boards). Have you tested? 

> Generally, this should only be called once the opponentGone event is received in the board game state stream and the claimWinInSeconds time has elapsed.

So you can get the number of seconds from the API. Please use that instead of 8 then, if this does work as intended.

Reply:
Btw lets not use 8. We can directly get it from the API.

> Generally, this should only be called once the opponentGone event is received in the board game state stream and the claimWinInSeconds time has elapsed.

Please use `claimWinInSeconds` instead of 8 then, if this does work as intended.